node/peer.js

```diff
@@ -340,12 +340,35 @@ TChannelPeer.prototype.countOutPending = function countOutPending() {
     return pending;
 };
 
+var QOS_UNCONNECTED = 0;
```

Comment:
We can change it going forward... however I'd rather stick with a non-string comparison, so how about instead sentinel objects with nice `.toString()` methods?